docs/source/new_project.rst

```diff
@@ -32,7 +32,7 @@ Copier will ask you questions for how to set up the project. These questions wil
      - The license type you want to use for this project. Options are MIT and BSD. For more information on these options see `Github's license page <https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository>`_.
    * - *What tooling would you like to use to enforce code style?*
      - A linter is a tool to automatically format for consistency (see :doc:`Linting <../practices/linting>`). We provide options for `black <https://black.readthedocs.io/en/stable/>`_, `pylint <https://pypi.org/project/pylint/>`_, or no linter. Choosing a linter will include it as a project dependency and include it in the :doc:`pre-commit <../practices/precommit>` hooks. Defaults to ``pylint`` during simple installation. 
-   * - *Do you want to use a tool to maintain a specific ordering for module imports?*
+   * - *Do you want to use a isort maintain a specific ordering for module imports?*
```

Comment:
I think the 'a' of 'use a tool' slipped in here

Reply:
Oops, fixed.